silvanadomnica_Task4/image_contrast.cpp

```diff
@@ -0,0 +1,39 @@
+#include <opencv2/highgui.hpp>
+#include <iostream>
+#include <opencv2/core/types.hpp>
+
+int main(int argc, char **argv)
+{
+    cv::Mat image;
+    image = cv::imread("Lenna.png", 1);
```

Comment:
you should use a constant instead of '1', one looking at your code won't know what it represents